peerconnection.go

```diff
@@ -1207,6 +1212,10 @@ func (pc *PeerConnection) drainSRTP() {
 				return
 			}
 
+			pc.mu.Lock()
```

Comment:
Should orphanSRTP have its own mutex? Not sure what this mutex is shared with and if it would cause contention/deadlocks.

Reply:
Fixed!